gesture_detection_main.py

```diff
@@ -6,5 +9,30 @@
 
 
 if __name__ == '__main__':
```

Comment:
If possible can we make a main() function that is called? I think that is a standard practice rather than code running in the if block.